### ISSUE TITLE
Centos7/OHPC libibverbs-devel not found

### ISSUE THREAD

**Issue by rajil** [easybuilders/easybuild#378]
I am trying to install python on a Centos7 machine and getting the following error:
```
$ eb Python-3.5.2-foss-2016b.eb -r
== temporary log file in case of crash /tmp/eb-I4sNmw/easybuild-2xSN_y.log
== resolving dependencies ...
ERROR: Failed to process easyconfig
/opt/ohpc/pub/libs/easybuild/3.3.1/software/EasyBuild/3.3.1/lib/python2.7/site-packages/easybuild_easyconfigs-3.3.1-py2.7.egg/easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.10.3-GCC-5.4.0-2.26.eb:
One or more OS dependencies were not found: [('libibverbs-dev',
'libibverbs-devel')]
```

The libibverbs-devel is installed on the system

```
# yum install libibverbs-devel
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
* base: mirror.tzulo.com
* epel: mirrors.kernel.org
* extras: dallas.tx.mirror.xygenhosting.com
* updates: centos.unixheads.org
Package rdma-core-devel-13-7.el7.x86_64 already installed and latest version
Nothing to do
```

**Comment by boegel** [comment]
@rajil Can you check the log file geenrated by EasyBuild and check whether you are hit by https://github.com/easybuilders/easybuild-easyconfigs/issues/4036?

You could be running into this problem if you have `XZ` installed already with an older version of EasyBuild (prior to EasyBuild v3.1.1).

If so, try to reinstall the `XZ` used as a dependency, for example using `eb XZ-5.2.2-foss-2016b.eb --force`, and see if that fixes the problem.

If not, it seems like the package has changed name, EasyBuild only checks whether `libibverbs-devel` is installed, not whether `rdma-core-devel` is installed.

You should be able to dance around this issue using `eb --ignore-osdeps ...`

**Comment by boegel** [comment]
Should be fixed by https://github.com/easybuilders/easybuild-easyconfigs/pull/5132

**Comment by boegel** [comment]
Fixed in EasyBuild v3.4.1 thanks to https://github.com/easybuilders/easybuild-easyconfigs/pull/5132
